Data/git_metadata.csv

```diff
@@ -0,0 +1,25 @@
+Repo,hash,branch
```

Comment:
I am not sure, this file should be part of the PR. It could break something else later on. For now, remove it from the PR. Thanks.

Reply:
I have removed this file from the latest PR.